apps/server/src/modules/deletion/controller/api-test/deletion-request-find.api.spec.ts

```diff
@@ -0,0 +1,92 @@
+import { ExecutionContext, INestApplication } from '@nestjs/common';
+import { Test, TestingModule } from '@nestjs/testing';
+import { ApiValidationError } from '@shared/common';
+import { Request } from 'express';
+import request from 'supertest';
+import { AuthGuard } from '@nestjs/passport';
+import { AdminApiServerTestModule } from '@src/modules/server/admin-api.server.module';
+import { EntityManager } from '@mikro-orm/mongodb';
+import { cleanupCollections } from '@shared/testing';
+import { DeletionRequestLogResponse } from '../dto';
+import { deletionRequestEntityFactory } from '../../entity/testing/factory/deletion-request.entity.factory';
```

Comment:
Please export this file with this factory function from `factory` dir (via `index.ts`) and then the whole factory from the `testing` dir (also via `index.ts`) so you'll end up with just `../../entity/testing` import.

Reply:
done